hotels.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>SEDONA Hotels</title>
+</head>
+
+<body>
+    <header>
+        <h1 class="visualli-hidden">SEDONA</h1>
+        <figure class="sedona-logo">
+            <img src="images/logo.png" alt="logo">
+        </figure>
+        <nav class="sedona-navigation">
+            <ul>
+                <li class="sedona-navigation_item"><a href="#">информация</a></li>
+                <li class="sedona-navigation_item"><a href="#">фото и видио</a></li>
+                <li class="sedona-navigation_item"><a href="#">карта штата</a></li>
+                <li class="sedona-navigation_item"><a href="#">гостиницы</a></li>
+            </ul>
+        </nav>
+    </header>
+    <aside class="sedona-hotels-choose">
+
+        <form action="get">
+            <section class="sedona-hotels_check">
+                <h2 class="sedona-hotels_check_heading">инфраструктура</h2>
+
+                <input type="checkbox" name="subscribe-field-swimming-pool" id="subscribe-field-swimming-pool" checked>
```

Comment:
You can use `<ul>` there. 
```
<ul> 
  <li>
     <input type="checkbox" name="subscribe-field-parking" id="subscribe-field-parking">
     <label for="subscribe-field-parking">парковка</label>
  </li>
.....
</ul>
```

Reply:
If it would be fields like password and email etc. Yes, you won't use ul/ol cause it's not a similar fields.